frontend/app/src/Root.tsx

```diff
@@ -21,9 +21,9 @@ export const Root = ({ children }: { children?: ReactNode }) => {
     try {
       return fetchUrl(CONFIG.CONFIG_URL);
     } catch (err) {
-      toast(() => (
+      toast(
```

Comment:
all these changes on `toast` makes an error on browser console + introduce flakiness on toast

Reply:
if you want to pass params, prefer making it explicit via props passing

Reply:
the warnings are addressed and I don't see where the flakiness can come from, what I can see now is that we don't wait correctly when accessing a page and before creating an object